src/main/java/uk/gov/ons/ctp/response/action/endpoint/ActionEndpoint.java

```diff
@@ -243,6 +245,29 @@ public ActionDTO updateAction(
     }
   }
 
+  /**
+   * PUT to re-run the aborted Actions for a specified actionId and to reschedule them .
+   *
+   * @param actionId Action Id of the Action/s to re run
+   * @return Success for the re ran actions
+   * @throws CTPException if update operation fails
+   */
+  @RequestMapping(value = "/rerun", method = RequestMethod.PUT)
+  public ResponseEntity<?> rerunAction(@RequestParam(value = "actionId") UUID actionId)
```

Comment:
if you want to get super fancy, actually you can make this a `List` of UUIDs and then you can specify multiple `actionId`s in the HTTP request, if for example there was a bunch of these you wanted to re-submit... it would save having to lööp.